CreeDictionary/CreeDictionary/templates/CreeDictionary/word-entries.html

```diff
@@ -31,7 +31,14 @@ <h2 class="definition-title definition-title--search-result">
                                         data-cy="linguistic-breakdown">{% if result.linguistic_breakdown_head %}
                                     {{ result.linguistic_breakdown_head|join:" + " }}
                                     + {% endif %}
-                                    <strong>{{ result.lemma_wordform.text }}</strong> + {{ result.linguistic_breakdown_tail|join:" + " }}</span>
+                                    <strong>{{ result.lemma_wordform.text|add:":" }}</strong>  <ol>                                    
```

Comment:
The `<ol>` should probably be on its own line, and be given a [BEM](http://getbem.com/naming/) class.

```suggestion
                                    <strong>{{ result.lemma_wordform.text|add:":" }}</strong>
                                    <ol class="linguistic-breakdown">                                    
```